blackbox/docs/best_practice/data_import.txt

```diff
@@ -218,11 +216,17 @@ For example: ``users.json``
 COPY FROM Command
 .................
 
+Upon execution, each node will check the provided path *locally* if the
+file exists and import the data it contains. Consequently this command will check
+``/tmp/best_practice_data/`` on each node in the cluster to import
+data from a file called 'users.json'.
+
```

Comment:
Imho it would be nice to also inform that if the file doesn't exist it won't return an error but instead the row count will be 0